tests/legacy-cli/e2e/tests/ivy/ivy-opt-out.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { replaceInFile } from '../../utils/fs';
+import { request } from '../../utils/http';
+import { killAllProcesses, ng } from '../../utils/process';
+import { createProject } from '../../utils/project';
+
+export default async function() {
```

Comment:
Very nice test to have. Will need to enhance it later with any further opt-out actions.